src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>MSBuild task (EvaluateProjectInfoForCodeGeneration) used by Microsoft.VisualStudio.Web.CodeGeneration.Tools</Description>
-    <TargetFrameworks>net461;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
I see this assembly ends up in `toolsassets/netcoreapp2.0`. It's ok to target `netcoreapp2.0` directly instead of ns2.0 as this assembly doesn't end up in a package.